### ISSUE TITLE
Turtlebot interactions

### ISSUE THREAD

**Issue by stonier** [turtlebot/turtlebot#143]
Interactions to be added to indigo turtlebot.

**Comment by stonier** [comment]
- **Documentation**
- [x] The turtlebot indigo wiki page
- [x] The turtlebot google+ page
- [x] The kobuki yujinrobot page
- [x] The turtlebot sig (google group)
- [x] Ros answers url with keyword search for turtlebot|kobuki|turtlebot2
- [ ] ... add others later as we fill in the documentation

**Comment by stonier** [comment]
**Admin** 

Better name for this role? These will need to be rocon_uri constrained so that they run on the qt remocon only (not the android remocon).
- [x] rocon_master_info
- [x] rqt_graph
- [x] rqt_console
- [ ] ...?

**Comment by stonier** [comment]
## Pairing

This will take a bit longer as we need to revise all the rapps in `turtlebot_rapps`. The rapps need their specification files upgraded and we should not call them 'android_xxx' anymore.

**Common Pairings**
- [x] rocon_apps/chirp + ''  (i.e. no remocon side app)

**Qt Remocon Pairings**
- [x] Make a Map : need to work out this one (need make a map rapp +  teleop, + rviz)
- [x] Navigation: navigation rapp + turtlebot_rviz_launchers/navigation.launch
- [x] Chatter : rocon_apps/talker + rocon_qt_gui/rocon_qt_listener
- [x] 3d sensing : need to make a rapp out of 3dsensor.launch + turtlebot_rviz_launchers/view_robot
- [x] Follower : follower rapp + "" (i.e. no remocon side app)

**Android Remocon Pairings**
- [x] Teleop
- [x] Make a map
- [x] Navigation
- [x] Panorama
- [ ] Map Store? (need to check)
- [x] Follower

**Comment by kentsommer** [comment]
All interactions are added except for Follower for Qt and Android. They are waiting on workflow discussion about how to handle launching with capabilities.

**Comment by stonier** [comment]
Map store we can update when we hit the android programs. Are we good to close this for now kent?

**Comment by kentsommer** [comment]
This is done, I will close it out for now. If there are any issues with the android pairings once all the android apps are updated I can open another issue.
